server/sonar-web/src/main/js/app/components/search/SearchShowMore.js

```diff
@@ -31,19 +32,20 @@ type Props = {|
   qualifier: string,
   selected: boolean
 |};
+*/
 
 export default class SearchShowMore extends React.PureComponent {
-  props: Props;
+  /*:: props: Props; */
 
-  handleMoreClick = (event: MouseEvent & { currentTarget: HTMLElement }) => {
+  handleMoreClick = (event /*: MouseEvent & { currentTarget: HTMLElement } */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)